cypress/support/config/settings.js

```diff
@@ -5537,23 +5282,7 @@ module.exports = () => ({
         },
         smoke: false,
       },
-      frontPage: {
-        environments: {
-          live: {
-            paths: ['/russian'],
-            enabled: true,
-          },
-          test: {
-            paths: ['/russian'],
-            enabled: false,
-          },
-          local: {
-            paths: ['/russian'],
-            enabled: false,
-          },
-        },
-        smoke: true,
-      },
+      frontPage: { environments: undefined, smoke: false },
```

Comment:
**Question for reviewer(s)**: Is it OK to have removed front page tests for Russian on live?

These will no longer run as part of the simorgh-cd pipeline checks (because `smoke: true`)

We have tests for serbian & zhongwen (which are still CPS frontpages on local, test & live)